chat_exporter/chat_exporter.py

```diff
@@ -210,8 +213,8 @@ async def produce_transcript(channel):
 
             if result_img:
                 cur_attach = await fill_out(channel, img_attachment, [
-                    ("ATTACH_URL", a.url),
-                    ("ATTACH_URL_THUMB", a.url)
+                    ("ATTACH_URL", a.proxy_url),
+                    ("ATTACH_URL_THUMB", a.proxy_url)
```

Comment:
Your reason for adding this was to stop the "?" image coming through? The issue you mentioned in the discord, wasn't that regarding embed footers? If so, then this wouldn't be a fix to that - as this is images standalone.

Reply:
This is a different issue. When images are deleted and then the transcript is made, they show as ?. Using the proxy url fixes this.

Reply:
Same issue exists with files too. Switching non images to proxy_url sometimes works, sometimes doesn't. Without proxy url you always get 0.00mb file that you cannot access.